unskript-ctl/unskript-client.py

```diff
@@ -1359,6 +1362,48 @@ def list_creds():
         index += 1
     print(tabulate(creds_data, headers='firstrow', tablefmt='fancy_grid'))
     
+def start_debug():
+    """start_debug Starts Debug session. This assumes that the remote
+       config is copied to /tmp as  /tmp/remoteconfig. If this file
+       does not exist, then this function will raise an exception.
+    """
+    if os.path.exists("/tmp/remoteconfig") is False:
```

Comment:
make the file path/name a parameter to CLI rather than hardcoding path.

Reply:
Done. Implemented `--start-debug --config <PATH_TO_FILE>`